types/recordrtc/recordrtc-tests.ts

```diff
@@ -0,0 +1,26 @@
+import RecordRTC from 'recordrtc';
+
+navigator.getUserMedia(
+    { audio: true, video: true },
+    stream => {
+        const instance = new RecordRTC(stream, {
+            type: 'video',
+            disableLogs: true,
+            bufferSize: 2048,
+        });
+
+        instance.stopRecording(() => {
+            const blob = instance.getBlob();
+        });
+    },
+    console.error,
+);
+
+const canvas = document.querySelector('canvas')!;
+
+const instance2 = new RecordRTC(canvas, {
+    type: 'canvas',
+});
```

Comment:
Do you need to address https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46333#issuecomment-663768359 where there are lot more missing properties. Let us know if thats something correct to add or can be ignored.
Thanks

Reply:
Those are internal properties which are not documented. Hence why I didn't add them to the definitions.